frontend/ui/src/components/Note/index.tsx

```diff
@@ -14,29 +14,33 @@ type NoteProps = {
 };
 
 const Note = ({note, onClick, removeNote, updateNote}: NoteProps): JSX.Element => {
-  return (
-    <div className={styles.note} onClick={onClick}>
-      <div className={`${styles.noteInner} ${onClick ? styles.clickable : ''} ${removeNote ? styles.isRemovable : ''}`}>
-        <div>
-          <span className={`${styles.noteNameExpanded}`}>{note.text}</span>
-          {removeNote && (
-            <span className={styles.removeNote} onClick={removeNote}>
-              <Close className={styles.closeButton} title="Delete" />
+  if (note.timestamp && note.text) {
```

Comment:
nitpick: Checking for the inverse and returning early makes for less indentation and better readability.